doc/source/conf.py

```diff
@@ -19,27 +19,39 @@
 # All configuration values have a default; values that are commented out
 # serve to show the default.
 
-import sys, os
+import sys
+import os
 
 import nibabel
 
+# Check for external Sphinx extensions we depend on
+try:
+    import texext
+except ImportError:
+    raise RuntimeError('Need to install "texext" package for doc build')
+try:
+    import numpydoc
+except ImportError:
+    raise RuntimeError('Need to install "numpydoc" package for doc build')
+
 # If extensions (or modules to document with autodoc) are in another directory,
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
 sys.path.append(os.path.abspath('../sphinxext'))
 
-# -- General configuration -----------------------------------------------------
+# -- General configuration ----------------------------------------------------
 
 # We load the nibabel release info into a dict by explicit execution
 rel = {}
-execfile('../../nibabel/info.py', rel)
+with open(os.path.join('..', '..', 'nibabel', 'info.py'), 'r') as fobj:
```

Comment:
Pretty sure this is only ever called in Sphinx, which does the path setting. A little disinclined to worry about checking the logic of relative path finding, but it shouldn't be too hard, if it seems like a good idea. (I wouldn't do `nibabel.__file__` in case somebody's "live" version of nibabel isn't the repository they're working in.)